### DISCUSSION TITLE
License problem reporting

### DISCUSSION THREAD

**Discussion by bastie** [community/community#22236] [New to GitHub]
<p>Hello,<br>
how can I report to GitHub a license problem in a project.</p>
<p>For example: GPL project is cloning and declared as Apache-2.0 conform like <a href="https://github.com/akkumar/jreversepro" rel="noopener nofollow ugc">https://github.com/akkumar/jreversepro</a></p>
<p>Thanx</p>

**Comment by kingthorin** [comment]
<p>What do you expect GitHub to do? (Especially with a project thats been idle for 11+ years.)</p>
<p>Why not just open an issue requesting clarification/change from the author/owner.</p>

**Comment by bastie** [comment]
<p>I looking for an answer for my question, how to do… - I don’t want to search for an actual project with issue. Also, what is todo if project owner not change or answer?</p>
<p>thx for your response</p>

**Comment by kingthorin** [comment]
<p>It’s not a problem for GitHub, they’re not a license enforcement organization.</p>

**Comment by bastie** [comment]
<p>Thanks - the license check and viral license problems come so over developer. Not nice!</p>

**Comment by kingthorin** [comment, marked as answer]
<p>You can report GPL license violations via: <a href="https://www.gnu.org/licenses/gpl-violation.en.html" rel="noopener nofollow ugc">https://www.gnu.org/licenses/gpl-violation.en.html</a></p>

**Comment by FormerlyChucks** [comment]
<p>You do realize that GH recognizes files like “license” or “license.md”, but not “LICENSE-2.0.txt”, right? It isn’t a “problem”, rather the project’s creator didn’t put a recognizable file name in the repo. Nothing is wrong here.</p>

**Comment by kingthorin** [comment]
<p>I believe the problem the original poster was getting at is that the software states it is GPL (readme etc) but then the repo has an APL 2 license file.</p>
